### PR TITLE
feat: added product page

### PR DESCRIPTION
**Pull request by keshavkumar-in**

### CODE REVIEW

File: products/index.html [hunk 1]
```diff
@@ -0,0 +1,229 @@
...
+                                    <button class="button button-success button-floating">
+                                        <i class="fa-solid fa-cart-plus"></i>
+                                    </button>
+                                </div>
+                            </div>
+                        </div>
+                        <div class="card card-vertical card-image border-radius-md card-shadow">
+                            <div class="card-header">
+                                <span class="badge badge-success-light badge-pill">SALE</span>
+                                <img src="/assets/images/products/Alex_maquine_shoulder.jpg" alt=""
```
Comment (animeshgarai09) [inline products/index.html:118]: Fill in alt attribute

File: index.html [hunk 2]
```diff
@@ -5,7 +5,7 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Wrixby UI- An E-Commerce Store</title>
+    <title>Wrixby ECom- An E-Commerce Store</title>
```
Comment (animeshgarai09) [inline index.html:8]: Tittle naming can be improved and also make it consistent for different pages.

**Review by animeshgarai09 — APPROVED** [review]
The code is superbly clean. Good to merge

**Review by hsnice16 — APPROVED** [review]
Nice work. You have used good names for the classes, maintained the space between declarations. You have used semantic HTML. 

Great yr, keep doing nice work.

File: products/index.html (hunk 1 above)
Comment (keshavkumar-in) [reply products/index.html:118]: Yeah Thanks for noting

File: index.html (hunk 2 above)
Comment (keshavkumar-in) [reply index.html:8]: Sure

**Comment by keshavkumar-in** [comment]
> Nice work. You have used good names for the classes, maintained the space between declarations. You have used semantic HTML.
> 
> Great yr, keep doing nice work.

Thanks!

**Pull request merged**